readthedocs/proxito/urls.py

```diff
@@ -86,6 +90,21 @@
     #     name='docs_detail',
     # ),
 
+    # External versions
+    # (requires to be before single version)
+    url(
+        (
+            r'^{DOC_PATH_PREFIX}external/html/(?P<project_slug>{project_slug})/'
```

Comment:
I do wonder about having the `project_slug` here. I know users will be annoyed with how ugly and long these URL's are, and we should be able to get the slug from the Domain. If we do that though, we end up with the issues around subprojects & translations, so maybe it's better to just serve all subprojects & translations as their own project slug on the domain?

Reply:
I don't have a good reply for this. I suppose we want to keep discussing around this before merging and going out with this approach.